packages/react-scripts/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fs/react-scripts",
-  "version": "4.4.0",
+  "version": "4.5.0",
```

Comment:
```suggestion
  "version": "5.0.0",
```

Reply:
Why no pin change?  Won't that not publish it?